tests/treactiveSigil.nim

```diff
@@ -0,0 +1,38 @@
+import sigils/reactive
+
+import unittest
+import std/sequtils
+
+suite "reactive examples":
+
+  test "reactive wrapper":
+
+    let
+      x = reactive(5)
+      y = computed[int]():
+        2 * x{}
+
+    check x.val == 5
+    check y.val == 0
+    x <- 2
```

Comment:
This is an invocation with the meaning "Set the value of x to 2" right? Wouldn't it make sense to also add a check for that ala `check x.val == 2` ?

Reply:
Hmmm, yeah I thought I did, but probably missed it